src/components/Layout/Layout.js

```diff
@@ -12,11 +13,27 @@ import useOffline from "hooks/useOffline";
 import "./_layout.scss";
 
 const Layout = ({ children }) => {
+  const [menuCollapsed, setMenuCollapsed] = useState(true);
+  const [sideNavCollapsed, setSideNavCollapsed] = useState(false);
+
   const [releaseNotification, setReleaseNotification] = useLocalStorage(
     "releaseNotification",
     false
   );
 
+  const location = useLocation();
+
+  // Check if pathname includes a model name - and then always collapse sidebar
+  const isModelNameSet = location.pathname.split("/models/")[1] !== "";
```

Comment:
This logic and state should probably be stored in the `ui` segment of the redux store so that it'll be easier to control with the new layout updates.

Reply:
After discussing offline this is ok to land as-is but it's known that it will likely have to change in the future.